app/models/transaction.rb

```diff
@@ -1,3 +1,9 @@
 class Transaction < ApplicationRecord
   belongs_to :invoice
+
+  validates :result, presence: true
+  validates :credit_card_number, presence: true
+  validates :credit_card_expiration_date, presence: true
+
+  enum :result, { failed: 0, success: 1 }
```

Comment:
I know this table wasn't included in our instructions for enums so mainly I want to keep in mind the question for instructors: Are they useful when there are only two results or would a boolean be better?